### PR TITLE
Operations guide

### PR DESCRIPTION
**Pull request by dswiecki**

### CODE REVIEW

File: docs/operations_guide/Operations.md [hunk 1]
```diff
@@ -0,0 +1,623 @@
...
+
+
+### Managing lifecycle of scenario
+
+State of the scenario can be viewed in the scenario list, in the scenario details view or v . Possible states can be grouped in the following categories:
+
+
+
+* Not running
+  * _NotDeployed _(initial status before first deploy)
```
[the remark this answers is not in the document]
Comment (dswiecki) [reply docs/operations_guide/Operations.md]: Done

File: docs/operations_guide/Operations.md [hunk 2]
```diff
@@ -0,0 +1,623 @@
...
+
+### Nussknacker operations REST API
+
+Nussknacker provides a few REST endpoints which may be helpful to check if everything is OK - e.g. can be used to implement health checks using standard monitoring tools.
+
+Below we describe endpoints that return general information about the Nussknacker state. Please see Deployment REST API section for endpoints allowing for management of a single scenario.
+
+
+
+* /api/app/buildInfo GET - returns basic information about deployed version of Nussknacker
```
[the remark this answers is not in the document]
Comment (dswiecki) [reply docs/operations_guide/Operations.md]: Done

File: docs/operations_guide/Operations.md [hunk 3]
```diff
@@ -0,0 +1,623 @@
...
+
+
+### Upgrading to new versions
+
+
+##### Upgrading Flink
+
+For patch version changes (e.g. 1.11.1 to 1.11.2) upgrade usually goes without problems - it’s enough to replace Flink with new version with rolling upgrade on cluster.
+
+For major release changes (e.g. 1.11 to 1.12) it’s a bit more involved, the safest way is the following:
```
[the remark this answers is not in the document]
Comment (dswiecki) [reply docs/operations_guide/Operations.md]: Done

File: docs/operations_guide/Operations.md [hunk 4]
```diff
@@ -0,0 +1,623 @@
...
+  "host": "0bda71c51449",
+  "additionalData": {}
+}
+```
+
+
+
+
+
+### Tuning scenario performance
```
Comment (ZbyszekMM) [inline docs/operations_guide/Operations.md:503]: We need an additional subsection on "managing Flink partitioning".  How does NK decide to assign key (keyBy argument) in the operators? Can we (using NK) affect how NK instructs Flink to choose the key? Does the key get changed during the scenario execution? 

It is only now that I understood why some fields in the Component configuration forms are called _keyBy_.  As a side effect of the aggregation window we (also) change(?) the key ... This cannot be a secret - although may be an advanced topic. BTW, we have agreed to change the keyBy label to groupBy label in the aggregate forms; I just detected this field in the Delay component. Is it somewhere else?

**Review by ZbyszekMM — APPROVED** [review]

**Pull request merged**